spec/lib/audit/catalog_to_moab_instance_spec.rb

```diff
@@ -199,6 +339,55 @@
         allow(PreservedObjectHandlerResults).to receive(:new).and_return(pohandler_results)
         c2m.check_catalog_version
       end
+
+      context 'check whether PreservedCopy already has a status other than OK_STATUS, re-check status if so' do
```

Comment:
same again?   shared example??  something.  It's tricky looking at all these and figuring out which cases group together and which don't, and so on.